packages/crew_profile_base.rb

```diff
@@ -0,0 +1,91 @@
+require 'package'
+
+class Crew_profile_base < Package
+  description 'Crew-profile-base sets up Chromebrew\'s environment capabilities.'
+  homepage 'https://github.com/chromebrew/crew-profile-base'
+  version '0.0.1'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/crew_profile_base-0.0.1-pre6-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/crew_profile_base-0.0.1-pre6-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/crew_profile_base-0.0.1-pre6-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/crew_profile_base-0.0.1-pre6-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '042a8b84951511fbc017b2418b8c1eb92d40cba81e1e4e00c0e1d903ad2af032',
+     armv7l: '042a8b84951511fbc017b2418b8c1eb92d40cba81e1e4e00c0e1d903ad2af032',
+       i686: 'c354988e590dbe7cb5d40f683c2a6ec8aecaf497d5844f75bc7a211690ab4406',
+     x86_64: '6ecb01e0cbee6a2bee0b8d86b5fc6d86db92aacf9e2ef2ff892b3d846d5bb4dc'
+  })
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/profile.d"
+  end
+
+  def self.postinstall
+    # Extract as module in postinstall to avoid creating package binaries.
+    # Turn off git warnings.
+    system 'git config --global advice.detachedHead false'
+    system 'git config --global init.defaultBranch main'
+    @git_dir = 'crew_profile_base_git'
+    @git_hash = version
+    @git_url = 'https://github.com/chromebrew/crew-profile-base'
+    FileUtils.rm_rf(@git_dir)
+    FileUtils.mkdir_p(@git_dir)
+    Dir.chdir @git_dir do
+      system 'git init'
+      system "git remote add origin #{@git_url}"
+      system "git fetch --depth 1 origin #{@git_hash}"
+      system 'git checkout FETCH_HEAD'
```

Comment:
Why are we doing this in postinstall instead of install?  And should this be the source instead of `/dev/null`?

Reply:
ask @satmandu, I don't quite understand this.

Reply:
It throws fewer error messages on install by avoiding `/dev/null`. It looks MUCH cleaner.